src/Symfony/Component/Routing/Loader/XmlFileLoader.php

```diff
@@ -133,6 +133,24 @@ protected function parseRoute(RouteCollection $collection, \DOMElement $node, $p
 
         list($defaults, $requirements, $options, $condition) = $this->parseConfigs($node, $path);
 
+        if (isset($requirements['_method'])) {
+            if (0 === count($methods)) {
+                $methods = explode('|', $requirements['_method']);
+            }
+
+            unset($requirements['_method']);
+            @trigger_error(sprintf('The "_method" requirement of route "%s" in file "%s" is deprecated since version 2.2 and will be removed in 3.0. Use the methods attribute instead.', $id, $path), E_USER_DEPRECATED);
```

Comment:
methods should be quoted as well: `Use the "methods" attribute instead.` (also in the  other lines).